modules/openapi-generator/src/main/resources/dart/libraries/dio/pubspec.mustache

```diff
@@ -14,6 +14,8 @@ environment:
 dependencies:
   dio: '>=4.0.1 <5.0.0'
 {{#useBuiltValue}}
+  one_of: '>=1.5.0 <2.0.0'
```

Comment:
@jaumard I have thought about this for sometime, and there are a couple of reasons why I decided to split this package apart:

1. The logic of these packages is `openapi-generator` independent, so it will work across major changes
2. Having it as a separate package will allow other generators with different serializers to use it, since `one_of` doesn't depend on `built_value`, so anyone can create a new package to support custom serializers (similar to [one_of_serializer](https://pub.dev/packages/one_of_serializer))

It's actually my bad that I forgot to document these packages, I was kinda rushing the PR and didn't care much about documentation nor licenses at the start.

Would it be ok to add all the Core contributors of OpenApi + The dart committee to these branches as maintainers ?

I can also add CI pipelines to github to auto publish the master branch to pub.dev, that should make the project completely independent of me

Reply:
I didn't have the courage to look directly at the source of your package ^^ cool if it can be used outside this scope. 

If there is a clear doc and license, that we can be maintainers and that publish is done by CI I have no more objections ^^ good for me at least :)

Reply:
I sent invitations to everyone in the dart committee, I also added MIT license file
I will be improving docs for the next couple of days